HW Day 2

```diff
@@ -0,0 +1,2 @@
+2) Runtime Errors; Because compile time errors are errors in syntax or variables. There are no errors like that but there are runtime errors. Run time errors are errors that accure while the programm is being executed.
+3)There are not mistakes
```

Comment:
hmm, review this over again - there are three mistakes, try and find them and let me know.